README.md

```diff
@@ -1,13 +1,23 @@
-# Project Mongo API
+# project-mongo-api
 
-Replace this readme with your own information about your project.
+## Backend
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+GET
 
-## The problem
+#### HOME:                 /
+#### GAMESLIST:            /games
+#### GAME BY ID:           /games/:id
+#### GAME BY TITLE:        /games?title= ADD A TITLE
+#### GAMES BY DEVELOPER:   /games?developer= ADD A DEVELOPER
+#### GAMES BY PUBLISHER:   /games?publisher= ADD A PUBLISHER
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+### View it live
 
-## View it live
+https://project-mongo-api-pi3gkxc3ea-lz.a.run.app/
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+
+## Frontend
+
+### View it live
+
+https://nes-games-mongodb.netlify.app/
```

Comment:
Very nice touch building a frontend! The visualisation of the content of the API is very helpful when trying to grasp what we are actually building in the backend!